### PR TITLE
Doc: Add proper installation guide links on README

### LINKED ISSUE #1909: Document the pip install for ansible-galaxy collection install arista.avd
Currently the ```ansible-galaxy collection install arista.avd``` doesn't install the python requirements. So it's better if we document another step to install the python requirements as well.

Solution: Use the github raw page to install the pip requirements

```pip
pip install -r https://raw.githubusercontent.com/aristanetworks/ansible-avd/devel/ansible_collections/arista/avd/requirements.txt
```

or for a specific version

```pip
pip install -r https://raw.githubusercontent.com/aristanetworks/ansible-avd/v3.6.0/ansible_collections/arista/avd/requirements.txt
```

### PR DESCRIPTION
**Pull request by tgodaA**
## Change Summary

Remove extra installation guidelines on github README and point to appropriate links.

## Related Issue(s)

Fixes #1909 

## How to test

Click on the new links added and check if the landing page is correct

### User Checklist

- N/A

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

File: ansible_collections/arista/avd/README.md [hunk 1]
```diff
@@ -68,46 +68,13 @@ This repository provides custom plugins for Ansible's collection __arista.avd__
...
-- cvprac
-- paramiko
-- jsonschema
-- requests
-- PyYAML
-- md-toc
+### Collection Installation
 
-**Ansible + Additional Python Libraries Installation:**
+Ansible galaxy hosts all stable version of this collection. Installation from ansible-galaxy is the most convenient approach for consuming `arista.avd` content. Please use the [github](docs/installation/collection-installation.md) or [avd.sh](https://avd.sh/en/devel/docs/installation/collection-installation.html) links.
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/README.md:74]: simplest is do just add one link as relative link. It should work on avd.sh too

File: ansible_collections/arista/avd/README.md [hunk 2]
```diff
@@ -68,46 +68,13 @@ This repository provides custom plugins for Ansible's collection __arista.avd__
...
-$ pip3 install -r requirements.txt
-```
-
-requirements.txt content:
+### Requirements
 
-```text
---8<-- "requirements.txt"
-```
+To install other requirements please use the [github](docs/installation/requirements.md) or [avd.sh](https://avd.sh/en/devel/docs/installation/requirements.html) links.
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/README.md]: Same here - just one link. And reword "To install requirements please use follow [this](link) guide"

**Comment by ClausHolbechArista** [comment]
plase also change the readme at the root of the repo

**Review by ClausHolbechArista — APPROVED** [review]
LGTM

**Review by emilarista — APPROVED** [review]
LGTM

**Pull request merged**